common/pipelinegen/config_builder.go

```diff
@@ -257,6 +257,14 @@ func insertServiceGraphPipeline(currentConfig *config.Config) {
 	}
 	pipeline.Exporters = append(pipeline.Exporters, consts.ServiceGraphConnectorName)
 	currentConfig.Service.Pipelines[rootPipelineName] = pipeline
+
+	// Add ServiceGraphConnectorName receiver to every prometheus pipeline starts with metrics/prometheus-
+	for pipelineName, pipeline := range currentConfig.Service.Pipelines {
+		if strings.HasPrefix(pipelineName, "metrics/prometheus-") {
+			pipeline.Receivers = append(pipeline.Receivers, consts.ServiceGraphConnectorName)
+			currentConfig.Service.Pipelines[pipelineName] = pipeline
+		}
+	}
```

Comment:
This will make it always enabled for any Prometheus user, right? 
Just making sure that this is what we want, as there is no way to disable it (or opt in), and any existing users will now get it after upgrade.

Do you think we should include it in any metrics destination going forward? or make it an opt-in configurable setting?